demo/src/app.js

```diff
@@ -128,6 +130,14 @@ const AppContent = () => {
         setEquipmentLabelling(labelling);
     };
 
+    const handleLanguageClick = (currentLanguage) => {
+        setCurrentLanguage(currentLanguage);
+        language =
+            currentLanguage === 'sys'
+                ? navigator.language.split(/[-_]/)[0]
```

Comment:
what if the sys language is Chinese ? should use English if language isn't supported.

Reply:
Yes we said we use english. I made the changes to have this behavior